File: java/strongly-typed-models/strongly_typed_models_retrieve.java
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("8d20758c-d74c-4f59-ae04-ee928c0816b7");

// Create strongly typed models according to https://kontent.ai/learn/strongly-typed-models
// Registers the model class for navigation items
client.registerType(Homepage.class);

CompletionStage<Homepage> homepageResult = client.getItem("hello_caas_world", Homepage.class);
// Use homepageResult
// homepageResult.thenAccept(homepage -> System.out.println(homepage.getHeadline())

File: net/strongly-typed-models/strongly_typed_models_retrieve.cs
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kontent.Ai.Delivery;
using Kontent.Ai.Delivery.Abstractions;
using KontentAiModels;

// Or register it through DI with services.AddDeliveryClient()
using var client = DeliveryClient.Create(new DeliveryOptions { EnvironmentId = "your-environment-id" }.UseProductionApi());

// Gets a content item by codename and maps it to the strongly typed model
var result = await client.GetItem<Homepage>("hello_caas_world").ExecuteAsync();

if (result.IsSuccess)
{
    var homepage = result.Value.Elements;
    // Use homepage
    // Console.WriteLine(homepage.Headline);
}